miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditStudyController.java

```diff
@@ -105,8 +106,23 @@ public Project populateProject(@PathVariable Long projectId) throws IOException
   public Collection<StudyType> populateStudyTypes() throws IOException {
     return studyService.listTypes();
   }
+  
+  @ModelAttribute("projects")
+  public Collection<Project> populateProjects() throws IOException {
+    return projectService.listAllProjects();
+  }
+
+  // put list of projects into Model object
+  @GetMapping(value = "/new")
+  public ModelAndView newStudy(ModelMap model) throws IOException {
+    User user = securityManager.getUserByLoginName(SecurityContextHolder.getContext().getAuthentication().getName());
```

Comment:
I believe we're trying to transition these long `securityManager` calls to `authorizationManager.getCurrentUser()`; @djcooke can you confirm? (I see there are a lot of residual securityManager calls in the Edit*Controller files!)

Reply:
Yeah, I change these calls into `authorizationManage#getCurrentUser` whenever I come across them. I haven't gone hunting them down though